test/processors/__snapshots__/TypeScriptInputProcessor.spec.ts.snap

```diff
@@ -0,0 +1,47 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`TypeScriptInputProcessor process() should be able to process input 1`] = `
+CommonInputModel {
+  "models": Object {},
```

Comment:
I dont see it generating any models 🤔

Reply:
The snapshot file doesn't save objects :confused:

Reply:
Ahhh right, cause we are mocking it 😅